### DISCUSSION TITLE
TypeORMStorage

### DISCUSSION THREAD

**Discussion by G0maa** [sequelize/umzug#667] [Ideas] [answered]
Hello Umzug Team,

I'd like a `TypeORMStorage` for TypeORM as opposed to `SeqeulizeStorage` for Sequelize.

I'm ok with contributing it to the repo, or as a separate `npm` package.

**Comment by mmkal** [comment, marked as answer]
I don't believe any of the maintainers are familiar with TypeORM, so I'd recommend a separate package and repo. I'd happily accept a pull request to link to it in the readme though.

**Reply by G0maa** [reply]
Ok, separate repo & package it is then.
